releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml

```diff
@@ -229,63 +233,81 @@ services:
     depends_on:
       - metadata
 
-  export-client:
-    image: nexus3.edgexfoundry.org:10004/docker-export-client-go:1.1.0
+  app-service-rules:
+    image: nexus3.edgexfoundry.org:10004/docker-app-service-configurable:master
```

Comment:
Should this image have a version like `1.1.0`?

Reply:
`master` for the images created from the new CI Pipelines. Eventually, all will be changed to `master` for nightly. When Fuji versions are created from nightly compose files, all the tags will get changed to the appropriate version number.

Reply:
Got cha